src/cljs/zetawar/views.cljs

```diff
@@ -384,7 +384,12 @@
                                  :on-hide hide-settings}
      [:> js/ReactBootstrap.Modal.Header {:close-button true}
       [:> js/ReactBootstrap.Modal.Title
-       (translate :configure-faction-title)]]
+       (str "Configure Faction: "
+            (when @faction
```

Comment:
I think I would turn this entire 'when' into a 'deftrack' named 'faction-color-name' in the 'subs' ns (accept conn + faction as args).